packages/language-support/src/tests/syntaxValidation/semanticValidation.test.ts

```diff
@@ -1174,7 +1174,7 @@ In this case, \`p\` is defined in the same \`MATCH\` clause as ((a)-[e]->(b {h:
     expect(getDiagnosticsForQuery({ query })).toEqual([
       {
         message:
-          'A pattern expression should only be used in order to test the existence of a pattern. It can no longer be used inside the function size(), an alternative is to replace size() with COUNT {}.',
+          'A pattern expression should only be used in order to test the existence of a pattern. It should therefore only be used in contexts that evaluate to a boolean, e.g. inside the function exists() or in a WHERE-clause. No other uses are allowed, instead they should be replaced by a pattern comprehension.',
```

Comment:
not for you to fix, but is it not odd for us to stop saying you'd need to use COUNT?